modules/rosa-cluster-classic/main.tf

```diff
@@ -26,7 +32,7 @@ resource "rhcs_cluster_rosa_classic" "rosa_sts_cluster" {
   }
   sts                = local.sts_roles
   aws_subnet_ids     = var.aws_subnet_ids
-  availability_zones = var.availability_zones
+  availability_zones = distinct(data.aws_subnet.provided_subnet[*].availability_zone)
   // aws_private_link   = var.aws_private_link
```

Comment:
remove those comments 
and keep the attribute of availability_zones

Reply:
Done